jsapp/js/account/organization/organizationQuery.ts

```diff
@@ -25,6 +42,22 @@ export enum OrganizationUserRole {
   owner = 'owner',
 }
 
+/**
+ * Mutation hook for updating organization. It ensures that all related queries
+ * refetch data (are invalidated).
+ */
+export function usePatchOrganization(orgUrl: string) {
```

Comment:
`orgUrl` seems like something that's a singleton and always available in the global state/cache, so let's simplify and grab it within the hook to avoid grabbing it in parent and passing around as parameter.

Reply:
@Akuukis I tried adding this previously:

```ts
const session = useSession();
const orgUrl = session.currentLoggedAccount?.organization?.url;
```

But `orgUrl` is not always `string` (TypeScript says so), so what's the best approach here?